embedded/embedded-test/src/test/resources/artifacts/apps/jdk-exported-resource-app/src/main/java/org/mule/test/ResourceGetter.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test;
+
+import java.io.InputStream;
+
+public class ResourceGetter {
+
+  public static InputStream getJdkResource() {
+    return Thread.currentThread().getContextClassLoader().getResourceAsStream("com/sun/xml/internal/messaging/saaj/soap/LocalStrings.properties");
```

Comment:
can you use another resource to test? i guess this test won't run as expected in a non-Oracle jvm